--- avogadro-1.1.1/libavogadro/src/pythonengine_p.h.orig	2014-10-30 11:15:36.000000000 +0100
+++ avogadro-1.1.1/libavogadro/src/pythonengine_p.h	2014-10-30 11:47:01.000000000 +0100
@@ -25,9 +25,13 @@
 #ifndef PYTHONENGINE_H
 #define PYTHONENGINE_H
 
+#ifndef Q_MOC_RUN
+#include <boost/python.hpp>
+#endif
+
 #include <avogadro/global.h>
 #include <avogadro/engine.h>
-#include <boost/python.hpp>
+
 
 namespace Avogadro {
 
--- avogadro-1.1.1/libavogadro/src/pythonextension_p.h.orig	2014-10-30 11:15:49.000000000 +0100
+++ avogadro-1.1.1/libavogadro/src/pythonextension_p.h	2014-10-30 11:46:35.000000000 +0100
@@ -26,10 +26,14 @@
 #ifndef PYTHONEXTENSION_H
 #define PYTHONEXTENSION_H
 
+#ifndef Q_MOC_RUN
+#include <boost/python.hpp>
+#endif
+
 #include <avogadro/extension.h>
 #include <avogadro/primitive.h>
 #include <avogadro/glwidget.h>
-#include <boost/python.hpp>
+
 
 #include <QWidget>
 #include <QList>
--- avogadro-1.1.1/libavogadro/src/pythoninterpreter.h.orig	2014-10-30 15:50:59.000000000 +0100
+++ avogadro-1.1.1/libavogadro/src/pythoninterpreter.h	2014-10-30 15:51:41.000000000 +0100
@@ -26,7 +26,9 @@
 #define PYTHONINTERPRETER_H
 
 #include <avogadro/global.h>
+#ifndef Q_MOC_RUN
 #include <boost/python.hpp>
+#endif
 #include <avogadro/primitive.h>
 #include <QString>
 
--- avogadro-1.1.1/libavogadro/src/pythonscript.h.orig	2014-10-30 15:53:16.000000000 +0100
+++ avogadro-1.1.1/libavogadro/src/pythonscript.h	2014-10-30 15:54:50.000000000 +0100
@@ -27,7 +27,9 @@
 #define PYTHONSCRIPT_H
 
 #include <avogadro/global.h>
+#ifndef Q_MOC_RUN
 #include <boost/python.hpp>
+#endif
 
 #include "pythonerror.h"
 
--- avogadro-1.1.1/libavogadro/src/pythontool_p.h.orig	2014-10-30 11:16:29.000000000 +0100
+++ avogadro-1.1.1/libavogadro/src/pythontool_p.h	2014-10-30 13:38:42.000000000 +0100
@@ -25,9 +25,13 @@
 #ifndef PYTHONTOOL_H
 #define PYTHONTOOL_H
 
+#ifndef Q_MOC_RUN
+#include <boost/python.hpp>
+#endif
+
 #include <avogadro/global.h>
 #include <avogadro/tool.h>
-#include <boost/python.hpp>
+
 
 #include <QObject>
 #include <QAction>
